### DISCUSSION TITLE
App-Filter _some and _none missing

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#24932] [Feature Requests]
### Describe the Bug

When filtering a o2m field the API allows to define if we'd folter for `_some` or `_none`.
In the App this is completely missing. 

Maybe related to #14228

### To Reproduce

Try to filter for a o2m field with a `_none` rule

### Directus Version

10.6.3

### Hosting Strategy

Self-Hosted (Custom)

**Comment by Dominic-Marcelino** [comment]
This could also be a discussion for a feature request, but as all other operators are available within the app this felt more like an issue / missing feature to me. Feel free to convert to a discussion you see it different :)

**Comment by rijkvanzanten** [comment]
> This could also be a discussion for a feature request, but as all other operators are available within the app this felt more like an issue / missing feature to me

Agreed! That also leaves me with the _Should I label this as a Bug or Improvement_ question :)

**Comment by rijkvanzanten** [comment]
As for implementation, when you select a O2M field, it should render similar to a group so we can toggle between the three states. Logical AND as example:

![CleanShot 2023-10-04 at 14 57 31@2x](https://github.com/directus/directus/assets/9141017/7e0fdc98-6b18-4c5d-a58d-0c2dfde6d676)

**Comment by gla23** [comment]
This feature has my vote!

I came across it while making my first Directus project. Directus did everything I needed it to do within the app (I could make bookmarks for some helpful queries) except that for my final query I needed  _none on a o2m. I ended up doing it through the REST API on a simple HTML doc, but it would have been really great for this to have been in the app itself.

**Comment by u12206050** [comment]
Done: 

<img width="573" height="270" alt="Screenshot 2025-11-02 at 12 52 10" src="https://github.com/user-attachments/assets/e23c4a0d-ec93-40cd-ae08-3a08aedecffa" />

**Comment by u12206050** [comment]
When clicking the Add Filter next to the none, only the subfields under Activity & Shift Registrations are available to select.
